modules/misc/news.nix

```
@@ -1436,6 +1436,22 @@ in {
           A new module is available: 'programs.ranger'.
         '';
       }
+
+      {
+        time = "2024-03-10T10:27:11+00:00";
+        message = ''
```

Could limit this so that it's only shown to people that have the module enabled:
```suggestion
        condition = config.services.gpg-agent.enable;
        message = ''
```